sdk/identity/identity/src/credentials/managedIdentityCredential/fabricMsi.ts

```diff
@@ -2,21 +2,42 @@
 // Licensed under the MIT license.
 
 import https from "https";
-import { createHttpHeaders, PipelineRequestOptions } from "@azure/core-rest-pipeline";
+import {
+  createHttpHeaders,
+  createPipelineRequest,
+  PipelineRequestOptions
+} from "@azure/core-rest-pipeline";
 import { AccessToken, GetTokenOptions } from "@azure/core-auth";
 import { MSI, MSIConfiguration } from "./models";
 import { credentialLogger } from "../../util/logging";
-import { mapScopesToResource, msiGenericGetToken } from "./utils";
+import { mapScopesToResource } from "./utils";
 import { azureFabricVersion } from "./constants";
 
+// This MSI can be easily tested by deploying a container to Azure Service Fabric with the Dockerfile:
+//
+//   FROM node:12
+//   RUN wget https://host.any/path/bash.sh
+//   CMD ["bash", "bash.sh"]
+//
+// Where the bash script contains:
+//
+//   curl --insecure $IDENTITY_ENDPOINT'?api-version=2019-07-01-preview&resource=https://vault.azure.net/' -H "Secret: $IDENTITY_HEADER"
+//
+
 const msiName = "ManagedIdentityCredential - Fabric MSI";
 const logger = credentialLogger(msiName);
 
+/**
+ * Formats the expiration date of the received token into the number of milliseconds between that date and midnight, January 1, 1970.
+ */
 function expiresInParser(requestBody: any): number {
   // Parses a string representation of the seconds since epoch into a number value
   return Number(requestBody.expires_on);
 }
```

Comment:
Same comment here as `expiresInParser`.

Something doesn't align here. The comment says "seconds since epoch" and the doc comment of the function says "milliseconds since". Which is the actual value and can we just do a `parseInt` where we need this function instead?

Reply:
There’s a default expires on parser function. This case is different. The comment should say “milliseconds since”, so I’ll update that.